impl/src/main/java/io/kroxylicious/testing/kafka/common/KafkaClusterConfig.java

```diff
@@ -103,6 +102,7 @@ public class KafkaClusterConfig {
             KRaftCluster.class,
             Tls.class,
             SaslPlainAuth.class,
```

Comment:
Do we need both SaslPlainAuth.class and SaslPlainAuth.List.class?  Isn't a single SaslPlainAuth just a list of one?

Reply:
I followed the approach that it was. We had an array of `UserPassword` and a test case with more than one user/password so that's why I did this List. It is the same to the `BrokerConfig.class` and `BrokerConfig.List.class` as well

Reply:
From experimentation, I see you are right.